### PR TITLE
viewing feature rspec fix2

### PR DESCRIPTION
**Pull request by irchik**
review need

### CODE REVIEW

File: spec/bvt/functionality/auth/password_resetting_spec.rb [hunk 1]
```diff
@@ -2,12 +2,13 @@
 
 feature "Password Resetting" do
   background "sign up user" do
-    sign_up_as(Gen::given_user_by_number(0))
+    user0 = Gen.user
```
Comment (romikoops) [inline spec/bvt/functionality/auth/password_resetting_spec.rb]: first problem is user0. What is it? I expect user1 and user2, or user and other_user
second is user0 is local variable in background. It is invisible in scenarios. You have to use @user instead here

File: spec/bvt/functionality/users/viewing_spec.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+feature "Viewing Users" do
+  scenario "User is viewing other user on user page" do
+    user1 = Gen.user
+    user2 = Gen.user
+    sign_up_as(user1)
+    sign_up_as(user2)
+    log_in_as(user2)
+    UsersPage.open.click_link_or_button user1.email
```
Comment (romikoops) [inline spec/bvt/functionality/users/viewing_spec.rb]: click_link_or_button is also bad method name. Link or button is ui element. And you have to think about business action, like view_user, navigate_to_user, open_menu, submit_data, fill_form. 

Also, I do not see where did you change it in UsersPage

File: features/functionality/auth/signup.feature [hunk 3]
```diff
@@ -1,22 +1,18 @@
...
 Feature: Sign Up
   As a user 
   I want to sign up to the system
   So I can login with a new account
   
-  @bvt
   Scenario: user can open sign up page via menu
     Given home page of web application
-    When I click sign up menu item on home page
+    When I click Sign up menu item on home page
```
Comment (romikoops) [inline features/functionality/auth/signup.feature]: Natasha already fixed that, please use lower case

File: features/functionality/auth/signup.feature [hunk 4]
```diff
@@ -1,22 +1,18 @@
...
     Then I should be redirected to sign up page
 
-  @bvt
   Scenario: visitor can initiate sign up
     Given login page of web application
-    When I click sign up menu item on login page
+    When I click Sign up menu item on login page
```
Comment (romikoops) [inline features/functionality/auth/signup.feature]: +1

File: pages/users_page.rb [hunk 5]
```diff
@@ -6,4 +6,8 @@ class UsersPage < WebPage
 
   include MainMenu
 
+  def click_user_view_link(user_email)
```
Comment (romikoops) [inline pages/users_page.rb]: bad method name, use logical name, not UI level. For instance, open_user

File: features/functionality/auth/signup.feature (hunk 3 above)
Comment (irchik) [inline features/functionality/auth/signup.feature]: fixed

File: features/functionality/auth/signup.feature (hunk 4 above)
Comment (irchik) [inline features/functionality/auth/signup.feature]: fixed

File: pages/users_page.rb (hunk 5 above)
Comment (irchik) [inline pages/users_page.rb]: fixed

**Pull request merged**